WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndTextFieldWithImageTableViewCell.swift

```diff
@@ -0,0 +1,126 @@
+import UIKit
+
+/// Contains a title label, a text field and an image.
+///
+final class TitleAndTextFieldWithImageTableViewCell: UITableViewCell {
```

Comment:
I was wondering if the image icon could be set to the cell's `accessoryView` so that we could reuse `TitleAndTextFieldTableViewCell`? (not sure if it's possible)

Reply:
Unfortunately not, since the `accessoryView` is not subject to constraints. The image ends up under the text field.

Reply:
ah got it, sounds difficult to customize layout with `accessoryView` 😞